src/inspectors/FileInspector.ts

```diff
@@ -4,6 +4,8 @@ import debug from 'debug';
 import { injectable, optional, inject } from 'inversify';
 import { Types } from '../types';
 import normalize from 'normalize-path';
```

Comment:
Isn't it unnecessary as you're using NodePath from @babel/core now?

Reply:
Not yet finished